most_active_mentors.py

```diff
@@ -0,0 +1,149 @@
+"""A module for measuring the number of very active mentors
+
+This module provides functions for measuring the number of active mentors on a project.
+
+This is measured by number of PR comments. We are working under the assumption that PR
+comments are left in good faith to move contributors further instead of nitpicking and
+dis-couraging them.
+
+Open questions:
+   - should there be an option to limit this to certain users, e.g. core maintainers?
+   - should there be a limit to how many comments per PR we consider to avoid having
```

Comment:
Yes, a limit makes sense to me. It may make sense to say that after 1 comment someone was involved with a PR, and after 3 comments they were heavily involved, and to stop counting after that. Those numbers are my personal thresholds in my mind but could be set to configurable numbers with reasonable defaults.